CHANGELOG.rst

```diff
@@ -1,6 +1,39 @@
 Change log
 ==========
 
+0.19.0 (2020-08-??)
+-------------------
+
+BREAKING CHANGES
+~~~~~~~~~~~~~~~~
+
+* **migrate to Python3.7**
+* **disable entry points and switch to ``python -m`` calls**
+* **rename ``_universum`` to ``universum``** *(configs need to be fixed)*
+
+New features
+~~~~~~~~~~~~
+
+* **github-handler:** add :doc:`a new Universum mode <args_github_handler>` to serve as a GitHub App
+* **pylint:** allow selecting exact python version
+* **nonci:** set project root to current directory
+* **github_vcs:** aquire GitHub token only when needed
+
+Bug fixes
+~~~~~~~~~
+
+* **swarm:** only update status for latest revisions
+* **nonci:** remove code report from nonci, launching steps twice
+* **report:** error message with successful build if not set to be reported
+* **setup:** make installing modules for GitHub VCS type non-mandatory
+* **docs:** limit nav panel depth to remove confusing subparagraphs
```

Comment:
In my opinion this is an internal or too minor fix.
Also, it is not in the product itself.